setup.cfg

```diff
@@ -1,13 +1,3 @@
-[nosetests]
-verbosity=2
```

Comment:
What do you think about running pytest with some verbosity? I usually find it useful.

Reply:
I normally set that when I'm trying to debug something and am testing locally. Happy to change the default if you think that's better.

Reply:
Yea, that'd be my preference.